examples/simple/posix/UB-PTHREAD6-bad.output

```diff
@@ -0,0 +1,7 @@
+Error: UB-PTHREAD6
+Description: Attempting to unlock a PTHREAD_MUTEX_DEFAULT or PTHREAD_MUTEX_NORMAL mutex which is not robust and which has not been locked by the current thread.
```

Comment:
But the type of the error is Undefined behavior.  If this was a warning and the type potentially undesired behavior, then I would have raised no objections.